cmake_ext.py

```diff
@@ -35,7 +31,7 @@ def build_extension(self, ext):
         cmake_args += [f"-DCMAKE_BUILD_TYPE={cfg}"]
         build_args = ["--config", cfg]  # needed by some generators, e.g. on Windows
 
-        if self.compiler and self.compiler.compiler_type == "msvc":
+        if self.compiler.compiler_type == "msvc":
```

Comment:
If `self.compiler` is not set, you are already in big trouble...

Reply:
This was there because it is needed.